### ISSUE TITLE
this still works? april 8th 2021

### ISSUE THREAD

**Issue by juancaiman** [Phoenix124/scribd-downloader#25]
I managed to follow all the steps prior to the installation of scribd-downloader.

I'm using linuxmint and when i tried: $ pip install scribd-downloader, it says "bash: pip: command not found".
then I followed some advice online an changed pip with pip3, resulting "pip3 install scribd-downloader"
and then the error message changed to: " ERROR: Could not find a version that satisfies the requirement scribd-downloader
ERROR: No matching distribution found for scribd-downloader"

So now I'm just curious if this downloader is still working or not. or maybe I have to change other stuff in the command lines.
Thanks!

**Comment by nmcc1212** [comment]
have you tried running from source instead of using pip

e.g. clone the directory etc.

**Comment by nmcc1212** [comment]
i think the package has been removed from pypi.org

https://pypi.org/project/scribd-downloader/

**Comment by morachimo** [comment]
Yes, it was removed form PyPi but you can still install it from source.

**Comment by KaKi87** [comment]
Duplicate of #9

**Comment by 3xploiton3** [comment]
> Yes, it was removed form PyPi but you can still install it from source.

![image](https://github.com/Phoenix124/scribd-downloader/assets/19517680/5838ab2a-49aa-48cc-b9a6-1a1d5a0ec84b)

**Comment by Phoenix124** [comment]
Yes for documents, no for books and audiobooks (they moved to Everand, see #50). The package isn't on PyPI, so install it with `pip install git+https://github.com/Phoenix124/scribd-downloader.git`.
